docs/source/release/v6.4.0/diffraction.rst

```diff
@@ -30,14 +27,18 @@ Engineering Diffraction
 New Features
 ############
 
-- Add a loader for ILL SALSA instrument
+- Added a loader for the ILL SALSA instrument.
 
 Bugfixes
 ########
 
-- Add a parent to an instance of the engineering diffraction interface created by loading a project so that it's a child of the main window. This also resolves an occasional crash when loading a project containing a workspace that was plotted on the Fitting tab
-- fix :ref:`algm-Rebin2D` to allow input of :ref:`EventWorkspace <EventWorkspace>` without crashing
-- Fix bug in Fitting tab of Engineering Diffraction UI where default list of files contains all previously focused files rather than the last ones generated by the last focus action
+* Fixed two problems when loading a project that includes the :ref:`Engineering Diffraction interface<Engineering_Diffraction-ref>`:
+
+  * resolved a crash when a user loads a project containing a workspace that was plotted on the :ref:`Fitting tab <ui engineering fitting>`.
+  * minimizing the main Workbench window also minimizes the Engineering Diffraction interface created from the project load.
+
+* Fixed :ref:`algm-Rebin2D` to allow input of :ref:`EventWorkspace <EventWorkspace>` without crashing.
```

Comment:
When this line about Rebin2D was added in #33617 I think it was put in the wrong place. This looks like it should be in the Framework section to me